File: Python/fizzbuzz.py
number = 0
while number < 100:
  number = number +1
  if number%15 == 0:
    print("Fizzbuzz")
  elif number%3 == 0:
    print("Fizz")
  elif number%5 == 0:
    print("buzz")
  else:
    print(number)

File: JS/fizzbuzz.js
for(let number = 0; number < 101; number++){
    if(number%15 == 0){
        console.log("Fizzbuss");
    }else if(number%3 == 0){
        console.log("Fizz");
    }else if(number%5 == 0){
        console.log("buzz");
    }else{
        console.log(number);
    }
}